app/models/edition.rb

```diff
@@ -66,6 +66,31 @@ class Edition < ApplicationRecord
 
   MINIMUM_SCHEDULING_TIME = { minutes: 15 }.freeze
 
+  scope :find_current, ->(id: nil, document: nil) do
```

Comment:
It's strange to see this phrased as a scope, as the name contains a verb and I'm used to scopes reflecting traits of a collection. Would it make more sense for it to be a method?

Reply:
The advantage to this being a scope is that we can chain this like we do with `lock.find_current` and wouldn't work in the same way as `find` or `find_by` that also act as chainable scopes.

Reply:
Also fair enough :-).